extensions/amp-analytics/0.1/vendors.js

```diff
@@ -964,6 +964,59 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     'optout': '_gaUserPrefs.ioo',
   },
 
+  'iplabel': {
+    'requests': {
+      'collectorUrl': 'm.col.ip-label.net',
+      'endpoint': 'https://${collectorUrl}/coll/',
+      'onload': '${endpoint}?' +
+      'T=${trackerId}&' +
+      'm=' +
+      '2502|${navTiming(navigationStart)}|' +
+      '2508|${navTiming(domainLookupStart)}|' +
+      '2509|${navTiming(domainLookupEnd)}|' +
+      '2510|${navTiming(connectStart)}|' +
+      '2512|${navTiming(connectEnd)}|' +
+      '2514|${navTiming(responseStart)}|' +
+      '2515|${navTiming(responseEnd)}|' +
+      '2517|${navTiming(domInteractive)}|' +
+      '2520|${navTiming(loadEventStart)}' +
+      '&ts=${timestamp}' +
+      '&ua=${userAgent}' +
+      '&d=${ipldim}' +
+      '&i=${clientip}' +
+      '&d[1]=${customdim}' +
+      '&d[2]=${business}' +
+      '&d[3]=${abtesting}' +
+      '&d[4]=${infrastructure}' +
+      '&d[5]=${customer}' +
+      '&u=${urlgroup}' +
+      '&w=${availableScreenWidth}&h=${availableScreenHeight}' +
+      '&r=${documentReferrer}' +
+      '&l=${browserLanguage}',
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'onload',
+      },
+    },
+    'transport': {
+      'beacon': false,
```

Comment:
just to confirm that your server doesn't support `beacon` and `xhrpost`. AMP will handle fallback case if browser doesn't support them, so it's always recommended to support all.